projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "item.tetra.quick_latch": "快卸销",
+    "tetra.salvage_quick_latch_action_hammer.label": "回收",
+    "tetra.salvage_quick_latch_action_pry.label": "回收",
+    "item.tetra.metal_scrap": "金属废料",
+    "item.tetra.metal_scrap.description": "奇形怪状的金属，它们的并无用处或用途未知，等待你赋予它们全新的价值",
+    "tetra.salvage_metal_scrap_action.label": "回收",
+    "item.tetra.vent_plate": "隔热板",
+    "item.tetra.vent_plate.description": "连接到锻造锤或传输单元上能稍微提高效率",
+    "tetra.salvage_vent_plate_action.label": "回收",
+    "item.tetra.combustion_chamber": "燃烧室",
+    "item.tetra.combustion_chamber.description": "可以被连接到锻造锤上提升 1 级锤子的等级",
+    "tetra.salvage_combustion_chamber_action.label": "回收",
+    "item.tetra.planar_stabilizer": "平面稳定器",
+    "item.tetra.planar_stabilizer.description": "可以被连接到锻造锤上，让用锻造锤合成的主要组件略微增加属性",
+    "tetra.salvage_planar_stabilizer_action.label": "回收",
+    "item.tetra.stonecutter": "切石器",
+    "tetra.salvage_stonecutter_action.label": "回收",
+    "item.tetra.earthpiercer": "穿地器",
+    "tetra.salvage_earthpiercer_action.label": "回收",
+    "item.tetra.forged_unsettling": "这件古物发出令人反胃的振动，它看起来被做成了装在手柄上的样子",
+    "item.tetra.forged_description": "能在下雪群系的地下深处找到",
+    "item.tetra.tooltip_reveal": "在物品栏中显示相关的升级",
+    "item.tetra.tooltip_expand": "§7[§8shift§7] §8+",
+    "item.tetra.tooltip_expanded": "§8[§fshift§8] §f+",
+    "tetra.slot.sword/blade": "剑刃",
+    "tetra.slot.sword/hilt": "剑柄",
+    "tetra.slot.sword/guard": "护手",
+    "tetra.slot.sword/pommel": "剑柄圆头",
+    "tetra.slot.sword/fuller": "附层",
+    "tetra.slot.double/head_left": "头部§7（左）",
+    "tetra.slot.double/head_right": "头部§7（右）",
+    "tetra.slot.double/handle": "手柄",
+    "tetra.slot.double/binding": "绑定",
+    "tetra.slot.double/accessory": "附件",
+    "item.tetra.modular.settle_progress.label": "固定进度",
+    "item.tetra.modular.settle_progress_gain.description": "当物品在被使用时这个组件会被缓慢地固定下来，当被固定后它会提供额外的完整度。\n \n§7进度：§e%s§7/§e%s\n \n§7基础要求：§e%s\n§7耐久惩罚：§e%s",
+    "item.tetra.modular.settle_progress_cost.description": "当物品在被使用时这个组件会被缓慢地固定下来，当被固定后它的完整度消耗会下降。\n \n§7进度：§e%s§7/§e%s\n \n§7基础要求：§e%s\n§7耐久惩罚：§e%s",
+    "item.tetra.modular.settle_full.label": "完全固定！",
+    "item.tetra.modular.settle_full_gain.description": "这个组件被完全固定了，它正提供额外的完整度。",
+    "item.tetra.modular.settle_full_cost.description": "这个组件被完全固定了，它的完整度消耗下降了。",
+    "item.tetra.modular.settle_full_null.label": "完全固定",
+    "item.tetra.modular.settle_full_null.description": "这个组件简易而容易安装，它不需要再被进一步固定了。",
+    "tetra.improvement.settled.name": "固定",
+    "tetra.improvement.settled.description": "为了增加用处，这个组件被固定了。这能提供额外的完整度",
+    "tetra.settled.toast": "一个组件被固定了！",
+    "tetra.improvement.arrested.name": "无法固定",
+    "tetra.improvement.arrested.description": "这个组件由无法被固定的材料制成",
+    "tetra.improvement.workable.name": "易加工",
+    "tetra.improvement.workable.description": "这个组件由易于加工的材料制造，打磨的耗时更短",
+    "tetra.improvement.quality.name": "高质量",
+    "tetra.improvement.quality.description": "这个组件质量不错，它提供少量的伤害与采集效率增益",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
    "tetra.improvement.quality.description": "这个组件质量较高，它可以提供少量的伤害与采集效率增益",
```